recipes/pygnome/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: 14486990e9b02a7fc60d8139435ed837d62fe4710a0acc91e297c5b596b6fac4
+
+build:
+  skip: True  # [py<39 or py>312 or python_impl == 'pypy']
+  missing_dso_whitelist:
+    - "*/lib_gnome.dll"         # [win]
+    - "*/point_in_polygon.dll"  # [win]
+  number: 0
+  script:
+    - export CMAKE_GENERATOR=Ninja  # [unix]
+    - cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - netCDF4                             # [build_platform != target_platform]
+    - scikit-build-core                   # [build_platform != target_platform]
+    - cython                              # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - libcxx-devel  # [osx]
```

Comment:
You'll need to limit this to `<19`, because libcxx 19 cannot be parsed by clang 15 anymore.